fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/ContainerServiceImpl.java

```diff
@@ -61,10 +63,22 @@ public Container findOrCreate(final Session session, final String path) {
 
             if (node.isNew()) {
                 initializeNewObjectProperties(node);
+
+                final Node parent = node.getDepth() > 0 ? node.getParent() : null;
+
+                if (parent != null) {
+                    touch(parent);
+                    touchLdpMembershipResource(node);
+                }
+            }
+
+            if (node.isNew()) {
+                touch(node);
             }
 
             return new ContainerImpl(node);
         } catch (final RepositoryException e) {
+            LOGGER.info("Error: {}", e);
```

Comment:
This is an actual error, so it deserves `ERROR` level logging. Otherwise, I disagree with @awoods , I think it is helpful to have the stack trace, but it's not something I would care very much about. As long as whatever is logged gives enough info to take definite and immediate action so that the request can be reissued successfully.